projects/creatorr.md

```diff
@@ -0,0 +1,77 @@
+## Project Name
+Creatorr
+
+## Category 
+NFT, Dapp, IPFS hosting, Social Media, Streaming
+
+## Project Description
+The goal is for Creatorr to be a platform which helps create an incentivized content creation support system. Creatorr fans earn unique dynamic NFT tokens from their content creator in exchange for donations and support. The tokens are collectible, tradeable and/or sellable.
+
+## Project Status
+MVP functionality being developed. Next Milestone is private alpha testing.
+
+## Previews
+https://nameless-wave-72583.herokuapp.com/index.html
+
+## Target Audience
+Content creators, NFT collectors and anyone that wants to support content creators.
+
+## Rough estimated user base (if applicable)
+New to the space but Web2 analogies would be
+Patreon: 6 Million Patrons 2021
+Twitch : Peaks pre-covid 800k a day
+
+## Github repo
+https://github.com/demo-hub/creatorr
+
+## Website
+https://nameless-wave-72583.herokuapp.com/index.html
+
+## Docs
+https://www.youtube.com/watch?v=6JNUpGN_XpM
+
+## Team Info
+Introduce your amazing team - how many team members are working on this project and who are they?
+
+### Team Size  
+Now we have 6 team members. 
+
+### Team members  
+
+João Rodrigues, Technical Leader
+- 3 years of web development experience
+
+Rosalie Avalon van Beek, Marketing manager
+- Content creator
+- Community manager
+- Dog lover
+
+Andrew Rangel, Full-stack dev
+- 9+ years native mobile development experience 
+- 5 years of full stack web development experience 
+- Product management and social media 
+
+Thomas Ford, Back-end dev/Systems
+- Python, Solidity/Blockchain Dev
+- Systems Management 
+- NFT maniac,Roboticist,Dog Lover, Degen
+
+Jurjen Folkertsma, Designer/Front-end dev
+- Passion for design
+- User experience is key
+- Front end dev
+
+Nuno Monteiro, Front-end dev
+- Ability to quickly change gears and adapt to changes
+- Creative thinking
+- Out of the box ideas
+
+## How the community can engage
+Email: - info@creatorr.fan
```

Comment:
Hey! can you add a show-and-tell topic in the discussions forum? thanks!

https://github.com/ipfs/community/blob/master/README.md#ecosystem-projects

Reply:
Done

Reply:
Can you link the discussion here?

Reply:
Just need this linked here

Reply:
Sorry, I linked it now.